packages/rfq-loadtest/config-template.yaml

```
@@ -0,0 +1,69 @@
+    rpcUrl_Read: 'https://mainnet.base.org'
+    rpcUrl_Sim: 'https://mainnet.base.org'
+    rpcUrl_Write: 'https://mainnet.base.org'
+  480:
+    FastRouterAddr: '0x00cd000000003f7f682be4813200893d4e690000'
+    # urls used for Reading / Tx Simulation / Tx Submit -- respectively. Change to others as needed.
+    rpcUrl_Read: 'https://worldchain-mainnet.g.alchemy.com/public'
+    rpcUrl_Sim: 'https://worldchain-mainnet.g.alchemy.com/public'
+    rpcUrl_Write: 'https://worldchain-mainnet.g.alchemy.com/public'
+
+TEST_ROUTES:
+  480>10: 
```

yamllint

[error] trailing spaces

(trailing-spaces)